src/app/games/conspirateurs/ConspirateursMove.ts

```diff
@@ -16,9 +16,7 @@ import { MoveWithTwoCoords } from 'src/app/jscaip/MoveWithTwoCoords';
 
 export class ConspirateursMoveDrop extends MoveCoord {
     public static encoder: MoveEncoder<ConspirateursMoveDrop> =
-        MoveCoordEncoder.getEncoder(ConspirateursState.WIDTH,
-                                    ConspirateursState.HEIGHT,
-                                    (coord: Coord) => ConspirateursMoveDrop.of(coord).get());
+        MoveCoord.getEncoder((coord: Coord) => ConspirateursMoveDrop.of(coord).get());
```

Comment:
créer le .from à la place comme pour C6 ?

Reply:
renommé .of en .from pour la consistance

Reply:
MoveCoord.getEncoder(ConspirateursMoveDrop.from); du coup !